static/js/compose_actions.js

```diff
@@ -396,6 +396,10 @@ exports.quote_and_reply = function (opts) {
         // are prone to glitches where you select the
         // text, plus it's a complicated codepath that
         // can have other unintended consequences.)
+        if (!compose_state.replying_to_message(message)) {
+            var reason = i18n.t('Please check your message destination.');
+            notifications.notify_above_composebox(reason);
+        }
         textarea.caret(0);
```

Comment:
What's your thinking about doing the `caret(0)`?  In other circumstances, e.g. "reply mentioning user", we don't do that sort of reset.  But I think there's a good argument for how this is different, in that this is guaranteed to be a multi-line block.  We should put a comment explaining that, though.

Reply:
Ok, I'll add a comment.

Reply:
re-pushed with comment

Reply:
What exactly does this call do?